### PR TITLE
fix invalid loglevel on OIDC filter

### PR DESCRIPTION
**Pull request by Shikugawa**

### CODE REVIEW

File: test/filters/oidc/jwks_resolver_test.cc [hunk 1]
```diff
@@ -66,10 +66,10 @@ const char invalid_jwt_public_key_[] = R"(
         {
            "kty": "XYZ",
            "crv": "P-256",
-           "x": "unko",
-           "y": "unko",
-           "alg": "unko",
-           "kid": "fuck"
```
Comment (incfly) [inline test/filters/oidc/jwks_resolver_test.cc:72]: I learned what "unko" means just now 😄

File: src/filters/oidc/token_response.cc [hunk 2]
```diff
@@ -81,7 +81,7 @@ std::shared_ptr<TokenResponse> TokenResponseParserImpl::Parse(
   const auto status = ::google::protobuf::util::JsonStringToMessage(
       raw_string_piece, &message, options);
   if (!status.ok()) {
-    spdlog::info("{}: JSON parsing error: {}", __func__,
+    spdlog::warn("{}: JSON parsing error: {}", __func__,
```
Comment (incfly) [inline src/filters/oidc/token_response.cc:84]: why not `error` and for the rest?

File: test/filters/oidc/jwks_resolver_test.cc (hunk 1 above)
Comment (Shikugawa) [reply test/filters/oidc/jwks_resolver_test.cc:72]: 🤷‍♂

File: src/filters/oidc/token_response.cc (hunk 2 above)
Comment (Shikugawa) [reply src/filters/oidc/token_response.cc:84]: My consideration is just "system error" should be `error`. But here is not a system error.

Comment (Shikugawa) [reply src/filters/oidc/token_response.cc:84]: My consideration is just severe "system error" that won't be processed anymore should be `error`. But here is not so critical, and no problem to continue entire process.

**Comment by Shikugawa** [comment]
@incfly @lizan Could you approve this?

**Review by incfly — APPROVED** [review]

**Pull request merged**